### ISSUE TITLE
Elevated circulating tiglylglycine concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7577]
**Preferred term label:**
Elevated circulating tiglylglycine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)**
An increased concentration of tiglylglycine in the blood circulation.

**Parent term (use hpo.jax.org/app)**
Abnormal circulating carboxylic acid concentration HP:0004354

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
300438 
PMID:12696021  5/5

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
